pom.xml

```diff
@@ -19,6 +19,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
+        <log4j2.version>2.15.0</log4j2.version>
```

Comment:
Apache released 2.16. yesterday. Should we go with: 2.16? (https://logging.apache.org/log4j/2.x/)